README.md

```diff
@@ -0,0 +1 @@
+This is a bank managing application written in Python
```

Comment:
I am thinking that maybe we can do a really cool README for our project:
https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/
https://www.makeareadme.com/